tests/e2e/features/3DViewer.feature

```diff
@@ -0,0 +1,40 @@
+Feature: preview 3D model
+    As a user
+    I want to preview a 3D model
+    So that I can make sure this is the correct file (e.g. before sharing it with others)
+
+
+    Scenario: preview 3D model
+        Given the user has logged in with username "admin" and password "admin"
+        And the user has uploaded the following 3D models:
+            | model1.glb |
+            | model2.glb |
+
+        # preview 3D model
+        When the user opens the file "model1.glb" in the 3D model viewer
+        Then the 3D model will be display in the viewport
+        And the file name "model1.glb" will be shown in the topbar
+
+        # toggle between standard and fullscreen mode
+        When the user clicks on “Enter full screen mode”
+        Then the 3D model will be displayed in full screen mode
+        And the topbar won’t be visible
+        When the user clicks on “Exit full screen mode”
```

Comment:
In my opinion there are multiple issues with phrasing the step like that:
1. I always suggest not to use the word "clicks" because you might want to run the test on a mobile device, then it should be "taps" and anyway
2. Its tightly coupled to the implementation, if you change the description of the button you have to change the feature file.
```suggestion
        When the user exists the full screen mode
```

Reply:
Would it matter if the user exits full screen mode by selecting the control element or using ESC key? I guess the suggested wording leaves it open which of the two options is used.

Reply:
If you want to test both, write two tests steps